src/Workspaces/Remote/ServiceHub/Services/RemoteSymbolSearchUpdateEngine.cs

```diff
@@ -50,6 +51,28 @@ public Task UpdateContinuouslyAsync(string sourceName, string localSettingsDirec
             return serializedResults;
         }
 
+        private class ProgressService : ISymbolSearchProgressService
```

Comment:
For my edification, which process does this run in?  Is this to marshal results from OOP back to VS?  Is there a convention to understand which side is which?